boring/src/pkey.rs

```diff
@@ -281,6 +281,24 @@ where
         private_key_to_der,
         ffi::i2d_PrivateKey
     }
+
+    // This isn't actually PEM output, but `i2d_PKCS8PrivateKey_bio` is documented to be
+    // "identical to the corresponding PEM function", and it's declared in pem.h.
+    private_key_to_pem! {
```

Comment:
For your convenience:

https://github.com/signalapp/boring/blob/8e6f19aa9a787812ad9f412ea4f7156bc5f12705/boring/src/macros.rs#L37-L74